pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -83,7 +83,7 @@ def _extract_binaries(self, arch, sdkVersion, build):
         self.exec_cmd(cmd)
         if arch == 'amd64' or arch == 'arm64':
             tarballFilename = f"{self.sdk}-{sdkVersion}-darwin-{rarch}.tar.gz"
-            share_path = "mac_arm64" if arch == 'arm64' else "mac"
+            share_path = "mac_arm64" if arch == 'arm64' and int(self.assembly.split('.')[1])>=12 else "mac"
```

Comment:
Please use `util.isolate_major_minor_in_group` to determine the group version and explicitly check if it is greater than or equal to 4.12.

Reply:
updated